main/get.c

```diff
@@ -829,7 +830,8 @@ extern int cppGetc (void)
 				int next = getcFromInputFile ();
 
 				if (next == NEWLINE)
-					continue;
+					// continue; <-- this may exit returning the backslash, which si wrong.
```

Comment:
OK. Take a look at #853 .
There are four commits instead of two because of the upstream merges. But the diff is correct.